graphql/index.ts

```diff
@@ -0,0 +1,174 @@
+import * as amqp from 'amqplib'
+import { execute } from 'apollo-link'
+import { WebSocketLink } from 'apollo-link-ws'
+import axios from 'axios'
+import { DocumentNode, print } from 'graphql'
+import { SubscriptionClient } from 'subscriptions-transport-ws'
+import * as ws from 'ws'
+import { FIREWALL_ENDPOINT, GRAPHQL_ENDPOINT, TOKEN } from './constant'
+import { CREATE_BAN, GET_BANS, SUBSCRIBE_BAN } from './src/banQueries'
+import { CREATE_ROUTER } from './src/routerQueries'
+
+
+let id: string
+let channel: amqp.Channel
+
+/* Utils */
+const getWsClient = function(wsurl: string): SubscriptionClient {
+  const client = new SubscriptionClient(wsurl, {
+    reconnect: true,
+    connectionParams: {
+      headers: {
+        authorization: TOKEN
+      }
+    }
+  }, ws)
+  return client
+}
+
+const createSubscriptionObservable = (wsurl: string, query: DocumentNode, variables: queryContent) => {
```

Comment:
Type should begin with an Uppercase character

Reply:
done